### ISSUE TITLE
`gum file` path argument is ignored

### ISSUE THREAD

**Issue by Birdie0** [charmbracelet/gum#397]
**Describe the bug**
Documentation mentions path argument, but it seems always ignored.

```
Usage: gum file [<path>]

Arguments:
  [<path>]    The path to the folder to begin traversing ($GUM_FILE_PATH)
```

**To Reproduce**
1. Run `gum file somefolder`, `GUM_FILE_PATH="somefolder" gum file` (somefolder directory must exist) or `gum file /`.

**Expected behavior**
Traversing should start in provided directory.

**Actual behavior**
It starts at same directory command was running from.

**Desktop:**
 - OS: Win11, WSL2 (Ubuntu 20.04)
 - Version: 0.10.0

**Additional context**

**Comment by bwrst** [comment]
See #326

The issue is fixed but there has yet to be a new release.

You can downgrade to the last working version using these commands:
```bash
sudo apt install gum=0.9.0
sudo apt-mark hold gum
```

Then, when there is a new release, you can unhold the package using:
```bash
sudo apt-mark unhold gum
````

**Comment by Birdie0** [comment]
fixed in `0.11.0`, thanks
